script/test_methods.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
 
-source ./scripts/functions.sh
```

Comment:
Noticed while running `lint.sh` that this was plural while the dir was not so changed